### DISCUSSION TITLE
Welcome to free-python-games Discussions!

### DISCUSSION THREAD

**Discussion by late7** [contribyte/free-python-games#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by apex862** [comment]
Im thinking of making a car game, but I thought the car is getting as lonely as my crying cat pic.
anyway, I need help making a ai car that can maneuver through the roads to challenge the player.
here is the easy and hardest levels
easy
![level 1](https://user-images.githubusercontent.com/127241407/236771962-09a9be46-0124-4956-aa70-82da520178e3.png)
hard
![level](https://user-images.githubusercontent.com/127241407/236771990-1ee18918-5b81-4047-97f9-89f945e9561a.png)
HELP!
